tests/e2e/scale/noobaa/test_scale_endpoint.py

```diff
@@ -84,7 +84,8 @@ def test_scale_endpoint_and_respin_ceph_pods(
         self._assert_endpoint_count(desired_count=1)
 
         # Create s3 workload using mcg_job_factory
-        job = mcg_job_factory(custom_options=options)
+        for i in range(10):
+            exec(f"job{i} = mcg_job_factory(custom_options=options)")
```

Comment:
@tiffanyn108  Maybe we need a wrapper method to simply call the job 'n' times and we could use this method at multiple places depending on how we want to load the cluster.

Reply:
This can be enhanced in mcg_job_factory() function which including the option to create multiple FIO pods.
It can be done in a separate PR.